services/token_update.proto

```diff
@@ -155,4 +155,9 @@ message TokenUpdateTransactionBody {
      * transaction will resolve to TOKEN_HAS_NO_METADATA_KEY
      */
     Key metadata_key = 17;
+
+    /**
+     * Determines whether the system should check the validity of the passed keys for update.
+     */
+    TokenKeyValidation should_verify_key = 18;
```

Comment:
I would expect a yes/no-answer when a property is names like this. What do you think about renaming it to `key_verification_mode`?

Reply:
I get your point and I totally agree!